CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.46] - 2021-12-01
+### Changes
+- metrics: Add ComplianceSuite status gauge and alert
```

Comment:
Can you make the changelog as verbose as for the previous versions? If you want to release now, we can merge this PR and I can propose a patch later, that's fine too. But we should get into the habit of maintaining a readable changelog

Reply:
let's do this in a subsequent PR